it/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -61,8 +61,8 @@ SaveTheseValuesEncrypted = Utilizza il Gestore password per ricordare questi dat
 SaveThisValueObscured = Utilizza il Gestore password per ricordare questo valore.
 SaveThisValueEncrypted = Utilizza il Gestore password per ricordare questo valore.
 SaveThisPasswordObscured= Utilizza il Gestore password per ricordare questa password.
-SaveThisPasswordEncrypted = Utilizza Gestione password per ricordare questa password.
-xSaveTheseValuesObscured = Ricorda questi dati. I dati archiviati non sono protetti da password.
+SaveThisPasswordEncrypted = Utilizzare Gestione password per ricordare questa password.
+xSaveTheseValuesObscured = Ricorda questi dati. I dati archivati non sono protetti da password.
```

Comment:
CORRECTIONS NEEDED. 

There is a typo: please use "archiviati" instead of "archivati".